config-templates/ibm/ibm-vpc-block-csi-driver/changelog.json

```diff
@@ -3,25 +3,21 @@
         "5.0": {
             "status": "supported",
             "latest-revision": {
+                "revision": "8",
+                "release-date": "22 March 2023",
```

Comment:
Also lets change date to `21 March` so that we get 1 day extra time for any unknown razee build rollout issues

Reply:
Addressed